tools/run-clangcldocker.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+COMMAND=$@
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+MAINSOURCE=$SCRIPTPATH/..
+ALL_CROARING_FILES=$(cd $MAINSOURCE && git ls-tree --full-tree --name-only -r HEAD | grep -e ".*\.\(c\|h\|cc\|cpp\|hh\)\$" | grep -vFf clang-format-ignore.txt)
+
+if clang-format-17 --version | grep -qF 'version 17.'; then
+  clang-format-17 "$@"
+  exit 0
+elif clang-format --version | grep -qF 'version 17.'; then
+  clang_format=( clang-format )
+  clang-format "$@"
+  exit 0
+fi
```

Comment:
It looks like we don't pass `--style=file --verbose -i $ALL_CROARING_FILES` if we use clang-format locally? I think we should use the same arguments in all the cases

Reply:
Corrected.